### PR TITLE
Users fixes

### PR DESCRIPTION
**Pull request by unparalleled-js**
* Limits the columns for TABLE output for `list users` cmds.

### CODE REVIEW

File: src/code42cli/cmds/users.py [hunk 1]
```diff
@@ -46,7 +47,12 @@ def list_users(state, org_uid, role_name, active, inactive, format):
     if inactive:
         active = False
     role_id = _get_role_id(state.sdk, role_name) if role_name else None
-    df = _get_users_dataframe(state.sdk, org_uid, role_id, active)
+    columns = (
+        ["userUid", "status", "username", "orgUid"]
+        if format == OutputFormat.TABLE
+        else None
```
Comment (unparalleled-js) [inline src/code42cli/cmds/users.py:53]: None means all columns

**Comment by timabrmsn** [comment]
One unrelated thing I found while testing this change was we don't cleanly handle a bad orgUid. The service returns a 400 we could handle better:
```
Failure in HTTP call 400 Client Error: Bad Request for url: https://www.crashplan.com/api/User?orgUid=10&pgNum=1&pgSize=500 

response.text: [{"name":"SYSTEM","description":"Organization was not found"}]
```

File: tests/cmds/test_users.py [hunk 2]
```diff
@@ -61,6 +63,16 @@ def test_list_outputs_appropriate_columns(runner, cli_state, get_all_users_succe
     assert "modificationDate" in result.output
 
 
+def test_list_when_table_format_outputs_expected_columns(
```
Comment (timabrmsn) [inline tests/cmds/test_users.py:66]: Should we test the columns we're filtering out _aren't_ in the output?

**Comment by unparalleled-js** [comment]
> One unrelated thing I found while testing this change was we don't cleanly handle a bad orgUid. The service returns a 400 we could handle better:
> 
> ```
> Failure in HTTP call 400 Client Error: Bad Request for url: https://www.crashplan.com/api/User?orgUid=10&pgNum=1&pgSize=500 
> 
> response.text: [{"name":"SYSTEM","description":"Organization was not found"}]
> ```

Yeah @terimurie also point this out. I was thinking we should just add it to py42 and let it naturally propagate to the CLI

**Review by timabrmsn — APPROVED** [review]

**Pull request merged**